core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -678,6 +678,9 @@ class PartKeyLuceneIndex(ref: DatasetRef,
     // If configured and enabled, Multi-column facets will be created on "partition-schema" columns
     createMultiColumnFacets(partKeyOnHeapBytes, partKeyBytesRefOffset)
 
+    val schemaName = Schemas.global.schemaName(RecordSchema.schemaID(partKeyOnHeapBytes, UnsafeUtils.arayOffset))
+    luceneDocument.get().addField(Schemas.TypeLabel, schemaName)
```

Comment:
Nit pick, this is same as ``addIndexedField`` and while looking up  that the method in the code when someone is trying to understand the setup, this gets missed.  If this line is changed to ``addIndexedField`` that would be helpful.

Reply:
sure